cmd/server/flags.go

```diff
@@ -240,232 +240,232 @@ var flags = []cli.Flag{
 		},
 	},
 	cli.StringFlag{
-		EnvVar: "DRONE_GITHUB_GIT_USERNAME,WOODPECKER_GITHUB_GIT_USERNAME",
+		EnvVar: "WOODPECKER_GITHUB_GIT_USERNAME",
 		Name:   "github-git-username",
 		Usage:  "github machine user username",
 	},
 	cli.StringFlag{
-		EnvVar: "DRONE_GITHUB_GIT_PASSWORD,WOODPECKER_GITHUB_GIT_PASSWORD",
+		EnvVar: "WOODPECKER_GITHUB_GIT_PASSWORD",
 		Name:   "github-git-password",
 		Usage:  "github machine user password",
 	},
 	cli.BoolTFlag{
-		EnvVar: "DRONE_GITHUB_MERGE_REF,WOODPECKER_GITHUB_MERGE_REF",
+		EnvVar: "WOODPECKER_GITHUB_MERGE_REF",
 		Name:   "github-merge-ref",
 		Usage:  "github pull requests use merge ref",
 	},
 	cli.BoolFlag{
-		EnvVar: "DRONE_GITHUB_PRIVATE_MODE,WOODPECKER_GITHUB_PRIVATE_MODE",
+		EnvVar: "WOODPECKER_GITHUB_PRIVATE_MODE",
 		Name:   "github-private-mode",
 		Usage:  "github is running in private mode",
 	},
 	cli.BoolFlag{
-		EnvVar: "DRONE_GITHUB_SKIP_VERIFY,WOODPECKER_GITHUB_SKIP_VERIFY",
+		EnvVar: "WOODPECKER_GITHUB_SKIP_VERIFY",
 		Name:   "github-skip-verify",
 		Usage:  "github skip ssl verification",
 	},
 	cli.BoolFlag{
-		EnvVar: "DRONE_GOGS,WOODPECKER_GOGS",
+		EnvVar: "WOODPECKER_GOGS",
 		Name:   "gogs",
 		Usage:  "gogs driver is enabled",
 	},
 	cli.StringFlag{
-		EnvVar: "DRONE_GOGS_URL,WOODPECKER_GOGS_URL",
+		EnvVar: "WOODPECKER_GOGS_URL",
 		Name:   "gogs-server",
 		Usage:  "gogs server address",
 		Value:  "https://github.com",
 	},
 	cli.StringFlag{
-		EnvVar: "DRONE_GOGS_GIT_USERNAME,WOODPECKER_GOGS_GIT_USERNAME",
+		EnvVar: "WOODPECKER_GOGS_GIT_USERNAME",
 		Name:   "gogs-git-username",
 		Usage:  "gogs service account username",
 	},
 	cli.StringFlag{
-		EnvVar: "DRONE_GOGS_GIT_PASSWORD,WOODPECKER_GOGS_GIT_PASSWORD",
+		EnvVar: "WOODPECKER_GOGS_GIT_PASSWORD",
 		Name:   "gogs-git-password",
 		Usage:  "gogs service account password",
 	},
 	cli.BoolFlag{
-		EnvVar: "DRONE_GOGS_PRIVATE_MODE,WOODPECKER_GOGS_PRIVATE_MODE",
+		EnvVar: "WOODPECKER_GOGS_PRIVATE_MODE",
 		Name:   "gogs-private-mode",
 		Usage:  "gogs private mode enabled",
 	},
 	cli.BoolFlag{
-		EnvVar: "DRONE_GOGS_SKIP_VERIFY,WOODPECKER_GOGS_SKIP_VERIFY",
+		EnvVar: "WOODPECKER_GOGS_SKIP_VERIFY",
 		Name:   "gogs-skip-verify",
 		Usage:  "gogs skip ssl verification",
 	},
 	cli.BoolFlag{
-		EnvVar: "DRONE_GITEA,WOODPECKER_GITEA",
+		EnvVar: "WOODPECKER_GITEA",
 		Name:   "gitea",
 		Usage:  "gitea driver is enabled",
 	},
 	cli.StringFlag{
-		EnvVar: "DRONE_GITEA_URL,WOODPECKER_GITEA_URL",
+		EnvVar: "WOODPECKER_GITEA_URL",
 		Name:   "gitea-server",
 		Usage:  "gitea server address",
 		Value:  "https://try.gitea.io",
```

Comment:
```suggestion
```

We should show an error if the user wants Gitea and forgot the url.

Reply:
agree!